test/TestSwaplace.test.ts

```diff
@@ -553,6 +688,60 @@ describe("Swaplace", async function () {
           ),
         ).to.be.revertedWithCustomError(Swaplace, "InvalidAddress");
       });
+
+      it("Should revert when wrong amount of ethers are sent by the {acceptee}", async function () {
```

Comment:
It lacks a test where the acceptee successfully accepts sending the ether

Reply:
> It lacks a test where the acceptee successfully accepts sending the ether

It's quite ambiguous since the acceptee can either accept or send ether. Did you mean to refer to your comment `Should verify with expect if the value was transferred as expected` or did you mean to say an entire new test where the `acceptee` successfully sends ethers to the `owner`.

Reply:
I had no place to input my request since the test was non-existent. There are no tests in which the one sending the ether as `msg.value` is the `acceptee` and the swap is accepted. There is thought, a test where the accepted sends ether and it fails, which is the test I pointed out above. The test lacking is the successful version of this failed one.

Reply:
Perfect! Onto it